validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -106,7 +118,13 @@ def __init__(self,
         Returns:
             None
         """
-        self._block_cache = block_cache
+
+        self._block_manager = block_manager
+        self._block_store = block_store
+        self._chain_head_fn = None
+        self._block_validity_fn = None
+        self._block_validity_cache = _Cache(100)
```

Comment:
It looks like you implemented the capacity? Are you keeping this in the PR?

Reply:
No, I have a fix that I will push soon that undoes this cache. This was the bandaid "cache".